cfme/tests/cloud/test_provisioning.py

```diff
@@ -68,7 +69,8 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name, t
         parameter = request.param
         if parameter == 'tag':
             inst_args['purpose'] = {
-                'apply_tags': ('{} *'.format(tag.category.display_name), tag.display_name)
+                'apply_tags': Check_tree.CheckNode(
```

Comment:
:+1: Good catch, missed this one. Was particularly hard to find instances of these paths being filled!